azure-mgmt-eventhub/src/main/java/com/microsoft/azure/management/eventhub/CaptureDescription.java

```diff
@@ -150,4 +156,24 @@ public CaptureDescription withDestination(Destination destination) {
         return this;
     }
 
+    /**
+     * Get a value that indicates whether to Skip Empty Archives.
+     *
+     * @return the skipEmptyArchives value
+     */
+    public Boolean skipEmptyArchives() {
+        return this.skipEmptyArchives;
+    }
+
+    /**
+     * Set a value that indicates whether to Skip Empty Archives.
+     *
+     * @param skipEmptyArchives the skipEmptyArchives value to set
+     * @return the CaptureDescription object itself.
+     */
+    public CaptureDescription withSkipEmptyArchives(Boolean skipEmptyArchives) {
```

Comment:
We will need to expose this property in this file (linked to an example of how to expose it):
https://github.com/Azure/azure-libraries-for-java/blob/78e8ff2940eba34bc63f8e7be6807a377500f5c7/azure-mgmt-eventhub/src/main/java/com/microsoft/azure/management/eventhub/implementation/EventHubImpl.java#L207

Reason : The event hubs team has explicitly asked for this.